### PR TITLE
Fix string cmd empty value bug

### PR DESCRIPTION
**Pull request by qiuyesuifeng**
Fix empty value get/set bug. 
For example,
set a "" should return "OK"
get a should return ""
also include mget/mset cmd

### CODE REVIEW

File: pkg/service/main.go [hunk 1]
```diff
@@ -212,3 +212,12 @@ func getRandomHex(buf []byte) []byte {
 
 	return buf
 }
+
+func getKeys(args [][]byte) [][]byte {
```
Comment (ngaut) [inline pkg/service/main.go]: Please using a simpler way.
return append([][]byte(nil), args...)

Comment (ngaut) [inline pkg/service/main.go]: My mistake, please ignore it.

**Comment by ngaut** [comment]
LGTM

File: pkg/service/string.go [hunk 2]
```diff
@@ -50,7 +50,8 @@ func (h *Handler) Set(arg0 interface{}, args [][]byte) (redis.Resp, error) {
 		return toRespErrorf("len(args) = %d, expect = 2", len(args))
 	}
 
-	s, err := session(arg0, args)
+	keys := getKeys(args)
+	s, err := session(arg0, keys)
```
Comment (siddontang) [inline pkg/service/string.go]: I think we should check nil key in store, not server.

Comment (qiuyesuifeng) [inline pkg/service/string.go]: sure, it is really better to process in store layer. i will modify it later.

**Comment by qiuyesuifeng** [comment]
No use. Close it.

**Pull request closed without merging**